chia/wallet/wallet_state_manager.py

```diff
@@ -15,7 +17,10 @@
 from chia.consensus.coinbase import pool_parent_id, farmer_parent_id
 from chia.consensus.constants import ConsensusConstants
 from chia.pools.pool_puzzles import SINGLETON_LAUNCHER_HASH, solution_to_pool_state
-from chia.pools.pool_wallet import PoolWallet
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from chia.pools.pool_wallet import PoolWallet
```

Comment:
I probably wouldn't worry about the conditional import unless it is needed to avoid issues.  Was there a circular import hiding here without breaking stuff?

The typing import should go above in it's proper section though, if kept.

It looks like this applies to the rest of the files below.

Reply:
Oh, this is leftovers - this shouldn't be here.